### DISCUSSION TITLE
Question: do web core vitals cover all pages in the same domain? what happens with subdomains as well?

### DISCUSSION THREAD

**Discussion by tomasts248** [GoogleChrome/lighthouse#15592] [Q&A] [answered]
Hi, in the context of PageSpeed Core Vitals reported stats, do they cover all stats related to the domain in question/ subdomains as well?

**Comment by adamraine** [comment, marked as answer]
It does not cover subdomains, for example google.com has different CWV results that store.google.com:
https://pagespeed.web.dev/analysis/https-google-com/tgq046449b?form_factor=mobile
https://pagespeed.web.dev/analysis/https-store-google-com/uu09ejvsj2?form_factor=mobile

**Reply by tomasts248** [reply]
Thank you for clarifying
